runelite-client/src/main/java/net/runelite/client/plugins/zalcano/ZalcanoConfig.java

```diff
@@ -146,41 +158,53 @@ default Title helperTitle()
 	 * TODO: improve helper
 	 */
 	@ConfigItem(
-		keyName = "showSteps",
-		name = "Show Step",
-		description = "",
-		titleSection = "helperTitle",
-		position = 9,
-		hidden = true //hidden until fully functional
+			keyName = "showSteps",
+			name = "Show Step",
+			description = "",
+			titleSection = "helperTitle",
+			position = 10,
+			hidden = true //hidden until fully functional
 	)
 	default boolean showSteps()
 	{
 		return false;
 	}
 
 	@ConfigItem(
-		keyName = "showAoeZalcanoMineable",
-		name = "Zalcano Mineable",
-		description = "Highlights Zalcano if she is mineable.",
-		titleSection = "helperTitle",
-		position = 10
+			keyName = "showAoeZalcanoMineable",
+			name = "Zalcano Mineable",
+			description = "Highlights Zalcano if she is mineable.",
+			titleSection = "helperTitle",
+			position = 11
 	)
 	default boolean showAoeZalcanoMineable()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "highlightGolem",
-		name = "Highlight Golem",
-		description = "Highlights the Golem that Zalcano spawns in.",
-		titleSection = "helperTitle",
-		position = 11
+			keyName = "highlightGolem",
+			name = "Highlight Golem",
+			description = "Highlights the Golem that Zalcano spawns in.",
+			titleSection = "helperTitle",
+			position = 12
 	)
 	default boolean highlightGolem()
 	{
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "playerCountAtZalcano",
+			name = "Counts players amount at Zalcano cave",
```

Comment:
Your `name` property here looks more like a description, and I'm not too sure what the description property really is